client/state/themes/actions.js

```diff
@@ -133,32 +136,30 @@ export function receiveServerError( error ) {
  * Returns an action object to be used in signalling that a theme activation
  * has been triggered
  *
- * @param  {Object}  theme Theme received
- * @param  {Object}  site Site used for activation
+ * @param  {Number}  themeId Theme to be activated
+ * @param  {Object}  siteId Site used for activation
  * @return {Object}  Action object
  */
-export function activateTheme( theme, site ) {
+export function themeActivation( themeId, siteId ) {
```

Comment:
Ah, probably so it's more in line with `themeActivated` and stuff like `themeActivationFailed`? Well what about `themeActivate` and `themeActivateFailure`? (`failure` -- being a noun -- is more consistent with `success` too, and also with action name constants).